bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/configuration/ServerConfiguration.java

```diff
@@ -85,6 +89,12 @@ public void setUISchemaFolder(final String uiSchemaFolder) {
 
    public void setServerPort(final int serverPort) { this.serverPort = serverPort; }
 
+   @Override
+   public String toString() {
+      return "ServerConfiguration [workspaceRootURI=" + workspaceRootURI + ", uiSchemaFolderURI=" + uiSchemaFolderURI
+         + ", serverPort=" + serverPort + "]";
+   }
+
```

Comment:
Why do we need this?

Reply:
It is not strictly necessary but I found it useful during debugging and it also shouldn't do any harm. So if you don't have any strong objections I would argue to keep that :-)